integration/indexes_compat_test.go

```diff
@@ -565,18 +565,12 @@ func TestIndexesDropRunCommand(t *testing.T) {
 	t.Parallel()
 
 	for name, tc := range map[string]struct { //nolint:vet // for readability
```

Comment:
Ok, as we touched these tests, let's ensure that compat tests have `Compat` in test name. E.g. this one should be `TestIndexesCompatDropRunCommand`

Reply:
Renamed this, and other index tests like `TestIndexesCreate` to `TestIndexesCompatCreate`